packages/editor-ui/src/components/layouts/ResourcesListLayout.vue

```diff
@@ -396,6 +396,27 @@ export default defineComponent({
 			this.loading = false;
 			await this.$nextTick();
 			this.focusSearchInput();
+
+			if (this.filtersLength() > 0) {
+				this.hasFilters = true;
+			}
+		},
+		filtersLength(): number {
+			let length = 0;
+
+			this.filterKeys.forEach((key) => {
+				if (key === 'search') {
+					return;
+				}
+
+				length += (
+					Array.isArray(this.filters[key]) ? this.filters[key].length > 0 : this.filters[key] !== ''
+				)
+					? 1
+					: 0;
+			});
+
+			return length;
```

Comment:
this can be simpler
```suggestion
		hasAppliedFilters(): boolean {
			return !!this.filterKeys.find((key) => key !== 'search' && (Array.isArray(this.filters[key]) ? this.filters[key].length > 0 : this.filters[key] !== ''));
```